src/models.py

```diff
@@ -582,6 +587,7 @@ def __init__(self, args, sent_encoder, vocab):
         self.vocab = vocab
         self.utilization = Average() if args.track_batch_utilization else None
         self.elmo = args.elmo and not args.elmo_chars_only
+        self.use_bert = args.use_bert
```

Comment:
Do we have both `args.use_bert` and `args.bert_model`? Probably can skip the format and just bool this, as Sam suggested: `bool(args.bert_model)`

Reply:
We actually don't have bet_model, but we have bert_model_name. We can bool that, but just seems a little wrong that bert_model_name would be used to check that bert is being used in a lot of places as well as for its actual value (for getting the right tokenizer etc)

Reply:
That's true, but then we have two flags for the functionality of one. It's pretty standard to use an emptystring (or anything else for which `bool(foo) == False`) to disable a feature.

Reply:
Sorry Yada, in the interest of minimizing flags, I'm going to stick to just using `args.bert_model_name` for the few times we need to check if using BERT.